pcli/src/command/tx.rs

```diff
@@ -162,6 +165,24 @@ pub enum TxCmd {
     /// Currently, only zero-fee sweep transactions are implemented.
     #[clap(display_order = 990)]
     Sweep,
+
+    /// Perform an ICS-20 withdrawal.
+    #[clap(display_order = 1000)]
+    Ics20Withdrawal {
```

Comment:
The command name will get inferred from the enum variant, so maybe we should call it just `Withdraw`, and set the display order to 350, so it's slotted in after swaps but before voting?